virtcontainers/clh.go

```diff
@@ -676,7 +676,6 @@ func (clh *cloudHypervisor) getAvailableVersion() error {
 	}
 
 	// Remove 'v' prefix if has one
-	clh.Logger().Error("DEBUG :", versionSplit[0])
```

Comment:
Hah, and in the next patch you dropped it .... probably better if you clean that up to not have it in the first place.